### PR TITLE
copy changes from rhel7 dockerfile

### PR DESCRIPTION
**Pull request by jasonbrooks**
fixes build error in #159

### CODE REVIEW

**Comment by felipevolpone** [comment]
@jasonbrooks why this helps fixing the issue in 159?

**Comment by jasonbrooks** [comment]
I saw [this comment](https://github.com/freeipa/freeipa-container/issues/159#issuecomment-330773245) mention that the build issue is likely due to changes between ip from 7.3 and ipa from 7.4, and that the rhel dockerfile contains those changes. I took the rhel dockerfile, changed it to start with centos as a base, and it did build.

File: Dockerfile.centos-7 [hunk 1]
```diff
@@ -1,18 +1,25 @@
-# Clone from the CentOS 7
+# Clone from the RHEL 7
```
Comment (adelton) [inline Dockerfile.centos-7]: This line probable shouldn't get changed. ;-)

**Comment by adelton** [comment]
@jasonbrooks, could you perhaps update the commit message to make it clearer in the `git log` output that we do this because CentOS now released RHEL 7.4 content?

**Comment by felipevolpone** [comment]
Tested and it works properly, fixing #159.

**Comment by stlaz** [comment]
@jasonbrooks Could you please perform the simple changes so that we can merge this PR?

**Comment by jasonbrooks** [comment]
@stlaz How does that look?

**Comment by stlaz** [comment]
@jasonbrooks You should probably also change the line in Dockerfile.centos-7 @adelton pointed out :)

**Comment by jasonbrooks** [comment]
@stlaz Thanks, I'd made the change locally but didn't add it to my commit

**Comment by stlaz** [comment]
Happens :) Thanks for your contribution, merging!

**Pull request merged**